src/libkomwm.py

```diff
@@ -736,7 +760,7 @@ def rule_sort_key(dict_):
                                 store_visibility(cl, 'line', st.get('object-id'), zoom)
                             for i in st.get('casing-dashes', st.get('dashes', [])):
                                 dr_line.dashdot.dd.extend([float(i)])
-                            addPattern(dr_line.dashdot.dd)
+                            addPattern(patterns, dr_line.dashdot.dd)
```

Comment:
It appeared that dash patterns are loaded from `patterns.txt` and extended with patterns from `*.mapcss` files. After that patterns values are exported back to `patterns.txt`.